llap-server/src/test/org/apache/hadoop/hive/llap/daemon/impl/TestLlapLoadGeneratorService.java

```diff
@@ -40,7 +44,32 @@ public void testLoadGenerator() throws InterruptedException, UnknownHostExceptio
 
     service.init(conf);
     service.start();
-    Thread.sleep(10000);
+    Thread.sleep(5000);
```

Comment:
Suggestion: 5s seems very long for the threads/service to come up. In the interest of having shorter test execution times, could this be (safely) reduced?

Reply:
Removed some of the sleeps and shortened where it is possible.